kurs/lab_4/task_4_1.py

```diff
@@ -0,0 +1,31 @@
+"""
+Perfect numbers generator
+Given the number, the task to return n perfect numbers.
+In number theory, a perfect number is a positive integer that is equal to the sum of its positive divisors,
+excluding the number itself. For instance, 6 has divisors 1, 2 and 3 (excluding itself), and 1 + 2 + 3 = 6,
+so 6 is a perfect number. More about perfect numbers read on https://en.wikipedia.org/wiki/Perfect_number.
+Note: Use generator for that task
+Input: 3
+Output: 6, 28, 496
+"""
+
+perfect_num = (6, 28, 496, 8128, 33550336, 8589869056, 137438691328, 2305843008139952128,
+               2658455991569831744654692615953842176, 191561942608236107294793378084303638130997321548169216)
+
+try:
+    col_num = int(input("Input :"))
+except:

```

Comment:
do not use bare exception: https://stackoverflow.com/questions/54948548/what-is-wrong-with-using-a-bare-except